btsieve/config/bitcoin_only.toml

```diff
@@ -1,8 +1,10 @@
 [bitcoin]
 node_url = "http://localhost:18443"
+zmq_endpoint = "tcp://127.0.0.1:28332"
+
+[bitcoin.authentication.basic]
 node_username = "bitcoin"
 node_password = "54pLR_f7-G6is32LP-7nbhzZSbJs_2zSATtZV_r05yg="
-zmq_endpoint = "tcp://127.0.0.1:28332"
 
```

Comment:
```suggestion
```

Reply:
I don't understand what you want me to do here :)

Reply:
Remove the double newline

Reply:
removed manually.